tests/pull-default-module/modules.yaml

```diff
@@ -7,6 +7,11 @@ data:
   version: 20180904161631
   context: cafebabe
   arch: x86_64
+  summary: Just a test module
```

Comment:
how come that now summary/description/license are mandatory? modulemd spec says it is optional.

Reply:
Nope, they've always been mandatory, see: https://github.com/fedora-modularity/libmodulemd/blob/master/spec.v2.yaml#L34

Reply:
They were not generating an error when using v1 API but they do in v2, that's the difference.

Reply:
That would have been a bug in v1. They should have been raising errors.